module-4/backend/week_one.md

```diff
@@ -5,17 +5,60 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+  I really enjoyed learning about non-relational databases with Neo4j and (FINALLY) implementing the Interactive Broker API. It was also very useful to get comfortable with writing python scripts without a web framework. I also took part of a datacamp course, which gave me a good understanding of how to create hacker data and interpret results.
```

Comment:
Love that you are working with all these other technologies, but this question was more specific to Turing intermission work on JS. This helps us improve the intermission work 😃